ajax/libs/balance-text/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "balance-text",
-  "version": "2.0.0",
-  "filename": "jquery.balancetext.min.js",
-  "description": "A jQuery plugin for implementing balancing of text across lines in a web page",
+  "version": "3.0.0",
```

Comment:
When the auto-updater in cdnjs update the lib, it will update version info. We don't need to modify it now.